src/ui_contextmenu.cpp

```diff
@@ -47,6 +51,9 @@ class OpenLyricsContextSubItem : public contextmenu_item_simple
                 case cmd_show_lyrics:
                 case cmd_manualsearch_lyrics:
                 case cmd_edit_lyrics:
+                case cmd_seek_to_next_lyric:
+                case cmd_seek_to_prev_lyric:
+                case cmd_seek_to_repeat_current_lyric:

```

Comment:
Hmm, to add to the reasons that it would be nice to not have to do the "get all panels, take one, check its lyrics" dance, we should disable these if there aren't any timestamped lyrics, and arguably we should disable the next/previous buttons if we're already on the last/first line respectively (or if not, make sure those buttons do sensible things in those cases, we don't want users to click a button that does nothing with no feedback).